prqlc/prqlc/src/semantic/resolver/functions.rs

```diff
@@ -131,19 +138,50 @@ impl Resolver<'_> {
                 name_hint: None,
                 args: vec![],
                 params: missing,
-                named_params: vec![],
                 body: Box::new(Expr::new(ExprKind::Func(inner_closure))),
-                return_ty: None,
-                env: HashMap::new(),
+
+                // these don't matter
+                named_params: Default::default(),
+                return_ty: Default::default(),
+                env: Default::default(),
+                generic_type_params: Default::default(),
```

Comment:
```suggestion
                ..Default::default()
```

?

Reply:
`Func` does not implement `Default`. It would be nice if Rust knew not all fields need the Default trait for this syntax to work, but it does not.

Reply:
Ah yes sorry...